### DISCUSSION TITLE
Clipboard: Paste text from anywhere

### DISCUSSION THREAD

**Discussion by RomanSuvorov** [expo/expo#15335] [Archived: Questions] [answered]
Hello,
Anybody know, how i can paste text in input from browser (as example) using Clipboard package on Android?

If i write 
```
      const text = await Clipboard.getStringAsync();
```
getting error 
```
Encountered an exception while calling native method: Exception occurred while executing exported method getStringAsync on module ExpoClipboard: Cannot convert argument of type class android.text.SpannableString
```

**Comment by prayaslashkari** [comment]
Any update on this?

**Reply by barthap** [reply]
Will be fixed in the next SDK 😀

**Comment by barthap** [comment, marked as answer]
This is fixed in SDK 45 (`expo-clipboard@3.0.1`) - now in `getStringAsync()` you can specify if you prefer to get plain text or HTML formatted - it should handle content from any app.
